docs/docs-content/audit-logs/monitor-with-datadog.md

```diff
@@ -0,0 +1,167 @@
+---
+sidebar_label: "Monitor Clusters with Datadog"
+title: "Monitor Host Clusters with Datadog"
+description: "Learn how to use Datadog to monitor Palette host clusters."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+hiddenFromNav: false
+tags: ["audit-logs", "datadog", "monitoring"]
+---
+
+You can use [Datadog](https://www.datadoghq.com/), a monitoring and analytics tool, to visualize the audit logs of your
+host clusters.
+
+This guide will walk you through the process of configuring Palette to export audit logs from your host clusters to
+Datadog.
+
+## Prerequisites
+
+- A [Palette account](https://console.spectrocloud.com) with tenant admin access.
+
+- A Datadog account. Visit the [Datadog official website](https://app.datadoghq.eu/signup) to create a Datadog account.
+
+- An active host cluster.
```

Comment:
For those that land on this page, let's link to the [Deploy a Cluster tutorial](https://docs.spectrocloud.com/tutorials/cluster-deployment/public-cloud/deploy-k8s-cluster/) in case they need help.

Reply:
:white_check_mark: